tests/test_commands.py

```diff
@@ -71,3 +72,13 @@ def test_flask_spec_indent(cli_runner, indent, tmp_path):
     else:
         assert f'{{\n{" " * indent}"info": {{' in stdout_result.output
         assert f'{{\n{" " * indent}"info": {{' in file_result.output
+
+
+def test_flask_spec_decimal_field(app, cli_runner):
+    @app.get('/qux')
+    @app.output(Qux)
+    def qux():
+        pass
+
+    result = cli_runner.invoke(spec_command)
+    assert 'openapi' in result.output
```

Comment:
I'm not sure if more assertions would be helpful here; this test was mostly intending to check if the command was successful.

Reply:
This is enough for now.